.github/workflows/gocept.amqprun.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9-dev]
```

Comment:
isn't 3.9 out now? so `-dev` is not required

Reply:
It is not yet available here. When it is available at GHA it can be used without the extra step of deadsnakes. but for now...